src/main/resources/static/api.yml

```diff
@@ -5462,6 +5501,26 @@ components:
         - translatedDocument
         - placementType
         - placementDates
+    PlacementApplicationDecisionEnvelope:
+      type: object
+      properties:
+        decision:
+          $ref: '#/components/schemas/PlacementApplicationDecision'
+        summaryOfChanges:
+          type: string
+        decisionSummary:
+          type: string
+      required:
+        - decision
+        - summaryOfChanges
+        - decisionSummary
+    PlacementApplicationDecision:
+      type: string
+      enum:
+        - accepted
+        - rejected
+        - withdraw
```

Comment:
What's the difference between `withdrawn` and `withdrawn`?

Reply:
Withdraw is if a new application is required, and withdrawn is if the PP has withdrawn the application. Will rename to be clearer 👍